### DISCUSSION TITLE
Swizzling, F-Strings, and Underscore Iterators/Variables

### DISCUSSION THREAD

**Discussion by durkisneer1** [godotengine/godot-proposals#7560] [Scripting]
It would be nice to be able to assign a Vector2 to a Vector3 such as `Vector3.xz = Vector2(x, y)` through swizzling, same for similar situations for other array types. F-Strings would make cleaner formatted string writing instead of the `text = "Score: %s" % score` mess. As for underscore variables, it would remove the need to use characters when writing `for` loops where the iterator/variable is unused. Ex: `for _ in range(len): print("hello")`

**Comment by AThousandShips** [comment]
See:
* https://github.com/godotengine/godot-proposals/issues/727
* https://github.com/godotengine/godot-proposals/issues/157
* https://github.com/godotengine/godot-proposals/issues/1282

Closing as these are all proposed, in the future please make *one* proposal for *one* feature, not multiple vert different features in one
